doc/_includes/inverse.rst

```diff
@@ -45,8 +29,13 @@ the maximum a posteriori (MAP) estimate under the following assumptions:
 - The measured data contain additive noise with a Gaussian distribution with a
   known covariance matrix. The noise is not correlated over time.
 
+Computing the inverse operator is accomplished using
+:func:`mne.minimum_norm.make_inverse_operator` and
+:func:`mne.minimum_norm.apply_inverse`. The use of these functions is presented
+in the tutorial :ref:`tut-inverse-methods`.
+
 The linear inverse operator
----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
MNE is not really a Bayesian approach as you just do a MAP on a probabilistic model. 
A Bayesian approach would involves some estimate of the posterior distribution and integrate something against is. I would rephrase `following from the Bayesian approach is given by the :math:`M \times N` matrix` to `following from regularized maximal likelihood of the above probabilistic model is given by the :math:`M \times N` matrix`.